mlos_core/mlos_core/tests/optimizers/optimizer_test.py

```diff
@@ -203,7 +206,8 @@ def objective(point: pd.DataFrame) -> pd.Series:   # pylint: disable=invalid-nam
 
         # loop for llamatune-optimizer
         suggestion = llamatune_optimizer.suggest()
-        assert suggestion['x'].iloc[0] == suggestion['y'].iloc[0]   # optimizer explores 1-dimensional space
+        _x, _y = suggestion['x'].iloc[0], suggestion['y'].iloc[0]
+        assert _x == _y or isclose(_x + _y, 3.)  # optimizer explores 1-dimensional space
```

Comment:
maybe use `pytest.approx()` instead?
```suggestion
        assert _x == pytest.approx(_y, 0.001)
```